src/pages/workspace/workflows/WorkspaceAutoReportingFrequencyPage.tsx

```diff
@@ -140,11 +131,14 @@ function WorkspaceAutoReportingFrequencyPage({policy, route}: WorkspaceAutoRepor
                         pendingAction={policy?.pendingFields?.autoReportingFrequency}
                         errors={ErrorUtils.getLatestErrorField(policy ?? {}, CONST.POLICY.COLLECTION_KEYS.AUTOREPORTING_FREQUENCY)}
                         onClose={() => Policy.clearPolicyErrorField(policy?.id ?? '', CONST.POLICY.COLLECTION_KEYS.AUTOREPORTING_FREQUENCY)}
+                        style={styles.flex1}
+                        contentContainerStyle={styles.flex1}
                     >
-                        <FlatList
-                            data={autoReportingFrequencyItems}
-                            renderItem={renderItem}
-                            keyExtractor={(item: WorkspaceAutoReportingFrequencyPageItem) => item.text}
+                        <SelectionList
```

Comment:
Coming from this https://github.com/Expensify/App/issues/47574
Since we don't pass shouldUpdateFocusedIndex we don't see selected background after selecting Submission frequency as monthly